common/net/conn/common.go

```diff
@@ -0,0 +1,51 @@
+package conn
+
+import (
+	"errors"
+	"math/rand"
+)
+
+// ErrWrongFormat - error when structure can't be parsed
+var ErrWrongFormat = errors.New("Unknown message format")
```

Comment:
```suggestion
var ErrWrongFormat = errors.New("unknown message format")
```
https://github.com/golang/go/wiki/CodeReviewComments#error-strings